### ISSUE TITLE
Clicking on Report button doesn't work.

### ISSUE THREAD

**Issue by PixelP0w3r** [gophish/gophish#1991]
What version of Gophish are you using?: v0.11.0

Brief description of the issue: When I click the report button next to username, it asks for confirmation but doesn't do anything except refresh the page.

What are you expecting to see happen? : The user statistic "reported" to increase by 1

What are you seeing happen? : Nothing/page refresh

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
time="2020-09-24T13:24:46-06:00" level=info msg="127.0.0.1 - - [24/Sep/2020:13:24:46 -0600] \"GET /api/campaigns/3?{} HTTP/2.0\" 200 59632 \"https://localhost:3333/campaigns/3\" \"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:80.0) Gecko/20100101 Firefox/80.0\""
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - Click the 'x' button under "reported" column next to a user in a campaign
 * Step 2 - When asking for verification click on "Continue"
 * Step 3 - Apparently nothing happens and the "Email Reported" statistic stays at 0

**Comment by glennzw** [comment]
Best to stick to one ticket, answered here:


https://github.com/gophish/gophish/issues/1979#issuecomment-698586674

**Comment by PixelP0w3r** [comment]
> 
> 
> Best to stick to one ticket, answered here:
> 
> [#1979 (comment)](https://github.com/gophish/gophish/issues/1979#issuecomment-698586674)

Hi glennzw. The comment you linked appears to be regarding a landing page. The issue I was reporting is that the "Reported" button doesn't seem to work to update the statistics. I am using the {{.URL}} parameter in my e-mail and the landing page works, but I cannot record, from the server end, users who report the phishing attempt to me.

**Comment by glennzw** [comment]
Sorry - I replies to the wrong post! Disregard

**Comment by jordan-wright** [comment]
@PixelP0w3r I recently resolved #1985 which I believe might be related to this. Can you confirm if this is related to your issue?

**Comment by PixelP0w3r** [comment]
> 
> 
> @PixelP0w3r I recently resolved #1985 which I believe might be related to this. Can you confirm if this is related to your issue?

That sounds exactly like my issue. Thanks for resolving so quickly.

**Comment by jordan-wright** [comment]
Good deal! Glad that fix works for you.
